packages/pancake-profile-sdk/src/functions/getProfile.ts

```diff
@@ -0,0 +1,95 @@
+import Cookies from "js-cookie";
+import { Profile, Nft } from "../types";
+import { getProfileContract, getPancakeRabbitContract } from "../utils/contractHelpers";
+import getTeam from "./getTeam";
+import nfts from "../constants/nfts";
+import getUsername from "./getUsername";
+
+const profileContract = getProfileContract();
+const rabbitContract = getPancakeRabbitContract();
+
+export type ProfileResponse = {
+  0: string;
+  1: string;
+  2: string;
+  3: string;
+  4: string;
+  5: boolean;
+};
```

Comment:
Can you check this response again. I think the response comes back with named keys as well (and I missed it).

Reply:
@hachiojidev seems to be only numbered keys:
```js
const profileResponse = await profileContract.methods.getUserProfile(address).call();
console.log(profileResponse);
// {
//   '0': '20301',
//   '1': '500',
//   '2': '1',
//   '3': '0xDf7952B35f24aCF7fC0487D01c8d569088888888',
//   '4': '34771',
//   '5': true
// }
```

Reply:
Okay, thanks. I asked that because I noticed in another response that it has numbered _and_ named keys.